package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "backbeat",
-  "version": "7.10.13",
+  "version": "7.10.14",
```

Comment:
Can you do the version bump in a separate commit

Reply:
Done.